robot/client/client.go

```diff
@@ -709,9 +713,8 @@ func (rc *RobotClient) PackageManager() packages.Manager {
 	return nil
 }
 
-// ResourceNames returns a list of all known resource names connected to this machine.
-//
-//	resource_names := machine.ResourceNames()
```

Comment:
Didn't know if this served a purpose -- happy to reintroduce if there's some magic documentation thing going on here